README.md

```diff
@@ -360,6 +361,90 @@ snet contract <ContractName> [--at AT] <functionName> PARAM1, PARAM2, ... [--tra
 
 ---
 
+```
+snet service init [--name NAME]
```

Comment:
These don't actually match the `metavar`s defined in arguments.py

Reply:
Fixed